docs/README.md

```diff
@@ -31,7 +31,7 @@
 | - | - | - |
 | PyPI | `pipx install sherlock-project` | `pip` may be used in place of `pipx` |
 | Docker | `docker pull sherlock/sherlock` | |
-| Debian family | `apt install sherlock` | Kali, Parrot, BlackArch, Testing, Sid |
+| Debian family | `apt install sherlock` | Kali, Parrot, Deb Testing, Deb Sid |
```

Comment:
```suggestion
| Debian family | `apt install sherlock` | Kali, Parrot, Debian and Sid |
| BlackArch | `pacman -S sherlock` |  |
```

Reply:
Works for me! Couldn't remember if BlackArch used pacman as well or if they used something special